index.js

```diff
@@ -53,8 +53,20 @@ module.exports = {
       'warn',
       {
         singleQuote: true,
-        printWidth: 999,
-      },
+        trailingComma: "all",
+        printWidth: 140,
+        bracketSameLine: true,
+        plugins: ["@serverless-guru/prettier-plugin-import-order"],
```

Comment:
Do we need to install this plugin?

Reply:
@imprashast  Would we need to install this plugin in this @warp-ds/eslint-config-repo, or is it enough that the repos that use the @warp-ds/eslint-config installs it as a devDep?

Reply:
This can be added to `peerDependencies` if you'd like :)
Basically anything this repo needs as a dependency in another repo using it should be added to this repo's peerDeps :)

Reply:
Does this solve anything that can't be solved with: https://eslint.org/docs/latest/rules/sort-imports ? 🙂

Reply:
Hmm good question! 🤔

Reply:
If I’ve understood it correctly, the `@serverless-guru/prettier-plugin-import-order` integrates with the Prettier formatter to automatically reorder imports during code formatting with minimal configurability to help maintain simplicity and consistency in style. While EsLint’s `sort-imports` offers more detailed configurability and enforces import order, which offers more control over code standards and styles. So I guess it depends on what we prioritize. If we want to prioritize code formatting with minimal setup, then it sounds like `@serverless-guru/prettier-plugin-import-order` could work for us, or if we want to prioritize style and quality enforcement then we could go with EsLint's `sort-imports`. If we’re already using EsLint for linting then maybe it is easier to incorporate ’sort-imports’, but it sounds like it is just a matter of preference. 🤔  @imprashast Correct me if I'm wrong here! 😄